lib/docs/filters/pytorch/clean_html.rb

```diff
@@ -2,9 +2,18 @@ module Docs
   class Pytorch
     class CleanHtmlFilter < Filter
       def call
+        breadcrumbs = at_css('.pytorch-breadcrumbs')
+        type_name = breadcrumbs.css('li')[1].content
+
```

Comment:
Suggestion:
Rather than sticking a temporary sentinel node within the HTML, instead change the order these two filters are executed. The order of execution for `clean_html` and `entries` filter is set on line 12 of `lib/docs/scrapers/pytorch.rb`

Reply:
Thanks for the suggestion!